tapalcatl_server/storage.go

```diff
@@ -120,6 +120,13 @@ func (s *S3Storage) Fetch(t tapalcatl.TileCoord, c Condition) (*StorageResponse,
 	return result, nil
 }
 
+func (s *S3Storage) healthCheck() (error) {
+	key := "sanity_check.txt"
```

Comment:
This key/file name should probably be configurable. It's not clear to me what the preferred method of configuration is for tapalcatl, so I'm leaving this here for now.